### DISCUSSION TITLE
Should grep lib be used? - error[E0603]: module `export` is private

### DISCUSSION THREAD

**Discussion by RyanBluth** [BurntSushi/ripgrep#1865] [Q&A] [answered]
I am trying to use v0.2.7 of the grep crate but get the following when running `cargo check`

```
error[E0603]: module `export` is private
   --> /Users/ryanbluth/.cargo/registry/src/github.com-1ecc6299db9ec823/grep-printer-0.1.5/src/stats.rs:11:39
    |
11  | #[cfg_attr(feature = "serde1", derive(Serialize))]
    |                                       ^^^^^^^^^ private module
    |
note: the module `export` is defined here
   --> /Users/ryanbluth/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.125/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Is there something that I'm missing? I've only included `grep = 0.2.7` in `Cargo.toml`

**Comment by BurntSushi** [comment, marked as answer]
What version of Rust and Cargo are you using?

Do you have any source code at all? Or is it just a blank project?

**Reply by RyanBluth** [reply]
I tried creating a new project and things worked fine, so I tried deleting my target folder and Cargo.lock file in my existing project and it resolved the issue. Not sure what was going on but I'm glad it's working now
